resources/public/api/conf/1.0/application.yaml

```diff
@@ -193,9 +181,12 @@ paths:
           application/json:
             schema:
               allOf:
-              - $ref: '#/components/schemas/individualPayeTestSupportSaTaxAnnualSummaryRequest'
-            example:
-              scenario: HAPPY_PATH_1
+                - $ref: '#/components/schemas/individualPayeTestSupportSaEmploymentsAnnualSummaryRequest'
+            examples:
+              happy-path-1:
+                $ref: '#/components/examples/taxRequestHappyPath1'
+              happy-path-2:
+                $ref: '#/components/examples/taxRequestHappyPath1'
```

Comment:
This should be `taxRequestHappyPath2`. Currently displaying happy path 1 content twice. Please update, thanks.